docs/long_description.md

```diff
@@ -0,0 +1,29 @@
+# Wagtail WordPress Import
+
+A package for Wagtail CMS to import WordPress blog content from an XML file into Wagtail.
+
+## Requirements
+
+1. Wagtail CMS Installed with initial setup
```

Comment:
```suggestion
1. Wagtail CMS Installed following the [initial setup](https://github.com/torchbox/wagtail-wordpress-import#wagtail-wordpress-import)
```

Although I'm not sure if we should be linking to here instead https://github.com/torchbox/wagtail-wordpress-import#initial-app-and-package-setup

Reply:
```suggestion
1. Wagtail CMS Installed following the [initial setup](https://docs.wagtail.io/en/stable/getting_started/index.html)
```
Actually that's a good point. The intention was the initial setup for a wagtail app.